manifests/config/apache.pp

```diff
@@ -200,6 +203,28 @@
     include ::apache::mod::intercept_form_submit
     include ::apache::mod::lookup_identity
     include ::apache::mod::auth_kerb
+  } elsif $keycloak {
+    apache::mod { 'auth_openidc':
+      package => 'mod_auth_openidc',
+    }
+
+
+    # This file is generated by keycloak-httpd-client-install and that manages
+    # the content. The command would be:
+    #
+    # keycloak-httpd-client-install --app-name ${keycloak_app_name} --keycloak-server-url $KEYCLOAK_URL --keycloak-admin-username $KEYCLOAK_USER --keycloak-realm ${keycloak_realm} --keycloak-admin-realm master --keycloak-auth-role root-admin --client-type openidc --client-hostname ${servername} --protected-locations /users/extlogin
+    #
+    # If $suburi is used, --location-root should also be passed in
```

Comment:
Not sure what `$suburi` is and where/how it can we used. Can you share some insights?

Reply:
Not sure how much it's used, but if `foreman_url` is passed in as `https://foreman.example.com/foreman` then `$suburi` will be set. Probably ok to ignore it for now.